### ISSUE TITLE
Show metadata for HEIC similar to JPG

### ISSUE THREAD

**Issue by BycreatinganaccountyouagreetotheTerms** [nomacs/nomacs#1175]
Currently the Metadata Ribbon panel (M) shows the below properties for JPGs:
Filename
Path
Size
Make
Model
Date Time
Exposure Time
FNumber
Flash
Focal Length

When opening a HEIC, only the first two are displayed.

Not sure if this should be considered a feature request or a bug.

**Comment by novomesk** [comment]
Can you attach a test HEIC file?

**Comment by BycreatinganaccountyouagreetotheTerms** [comment]
I have attached 2 files. The HEIC is the original from iPhone. I used a converter to get the JPG. Metadata is contained in both files but Nomacs does/can not display the HEIC metadata.

PS. I had to rename the HEIC as jpg as I couldn't upload it otherwise.

![IMG_0228 HEIC](https://github.com/user-attachments/assets/8c051153-1d09-4b56-8846-2c24e7dd0feb)
![IMG_0228](https://github.com/user-attachments/assets/254e6a46-c27d-4954-953d-8f5cc71c1c2c)

**Comment by BycreatinganaccountyouagreetotheTerms** [comment]
Let me try again.
[images.zip](https://github.com/user-attachments/files/17448312/images.zip)

**Comment by BycreatinganaccountyouagreetotheTerms** [comment]
I have attached 2 files. The HEIC is the original from iPhone. I used a converter to get the JPG. Metadata is contained in both files but Nomacs does/can not display the HEIC metadata.
[images.zip](https://github.com/user-attachments/files/17448313/images.zip)

**Comment by novomesk** [comment]
Do you see the info what I have?

![screenshot](https://github.com/user-attachments/assets/02b00778-d7a4-4abd-a048-21ad61e022ea)

What version of nomacs and what OS are you using?

**Comment by BycreatinganaccountyouagreetotheTerms** [comment]
For JPG, there is File, Exif, and Data headings. For HEIC there is only the File heading. I am using 3.16.1709 on Windows 10 22H2 19045.4894.
If I check for updates, it says 3.17.2~~~(text overflow) is available, but update doesn't seem to work.
If I go to https://github.com/nomacs/nomacs/releases, the latest non Beta release is 3.16.

**Comment by novomesk** [comment]
Try uninstall the 3.16 version and install
https://github.com/nomacs/nomacs/releases/tag/3.19.1

**Comment by BycreatinganaccountyouagreetotheTerms** [comment]
Thank you. I generally do not install beta software on production device. Also I am afraid of losing any customised settings in Nomacs by uninstalling and installing. However, I did install 3.19.1 on a separate device and it appears to work.

**Comment by leejuyuu** [comment]
As this works with the latest version, I am closing this as completed. Feel free to reopen if there's a further issue.
